docs/sources/installation/kubernetes.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Deploy on Kubernetes
```

Comment:
+++
title = "Deploy Grafana on Kubernetes"
description = "Guide for deploying Grafana on Kubernetes"
keywords = ["grafana", "configuration", "documentation", "kubernetes"]
weight = 700
+++